terraform/create_backend_vars.sh

```diff
@@ -11,9 +11,12 @@ fi
 ACCESS_KEY=$(echo "${S3_CREDENTIALS}" | jq -r '.access_key_id')
 SECRET_KEY=$(echo "${S3_CREDENTIALS}" | jq -r '.secret_access_key')
 REGION=$(echo "${S3_CREDENTIALS}" | jq -r '.region')
+BUCKET=$(echo "${S3_CREDENTIALS}" | jq -r '.bucket')
 
 {
   echo "access_key = \"$ACCESS_KEY\""
   echo "secret_key = \"$SECRET_KEY\""
   echo "region = \"$REGION\""
+  echo "bucket = \"$BUCKET\""
+  echo "prefix = \"$BUCKET\""
```

Comment:
```suggestion
  echo "prefix = \"dev\""
```

Reply:
If you want these code changes to be part of review, please commit prior to tabletop.